src/main/java/org/eclipse/golo/runtime/RegularMethodFinder.java

```diff
@@ -82,37 +86,39 @@ public boolean isOverloaded() {
   }
 
   private boolean isValidPrivateStructAccess(Method method) {
-    Object receiver = invocation.arguments[0];
+    Object receiver = invocation.arguments()[0];
     if (!(receiver instanceof GoloStruct)) {
       return false;
     }
     String receiverClassName = receiver.getClass().getName();
     String callerClassName = callerClass.getName();
-    return method.getName().equals(invocation.name)
-      && isPrivate(method.getModifiers())
-      && (receiverClassName.startsWith(callerClassName)
-          || callerClassName.equals(reverseStructAugmentation(receiverClassName)))
-      && TypeMatching.argumentsMatch(method, invocation.arguments);
+    return method.getName().equals(invocation.name())
+        && isPrivate(method.getModifiers())
+        && (receiverClassName.startsWith(callerClassName)
+        || callerClassName.equals(reverseStructAugmentation(receiverClassName)))
+        && TypeMatching.argumentsMatch(method, invocation.arguments());
   }
 
   private static String reverseStructAugmentation(String receiverClassName) {
     return receiverClassName.substring(0, receiverClassName.indexOf(".types"))
-      + "$" + receiverClassName.replace('.', '$');
+        + "$" + receiverClassName.replace('.', '$');
   }
 
 
   private Stream<Method> findInMethods() {
```

Comment:
Is it meaningful to extract `findInMethods`, `findInFields`, `toMethodHandles` and others into a Reflect/InvokeUtils ?